docs/source/quickstart.md

```diff
@@ -0,0 +1,37 @@
+(quickstart)=
+## Quickstart
+Robbie now uses Nextflow to manage the workflow and can be run on a local system or a supercomputing cluster. You can use a container via singularity, docker, or the host's software. The current development cycle tests Robbie using singularity on an HPC with the Slurm executor - other setups *should* work but haven't been extensively tested.
+
+### `images.txt`
```

Comment:
I think you should add a sentence that says:
By default, this file is called images.txt and is in your current directory. You can give Robbie a custom file name and location using `--image_file <path>/<file_name>.txt`

I think the next sentence may be clearer as:
For example, if there is a directory named "images" containing the `.fits` files:

Reply:
Changed it to:

>Before running Robbie, you will need to create a text file that contains the paths to each image to be processed. By default, this text file is called `images.txt` and is in your current directory. You can also give Robbie a custom file name and location using `--image_file <path>/<file_name>.txt`. For example, if there is a directory named "images" containing the `.fits` files:

Reply:
Perfect